src/index.ejs

```diff
@@ -74,91 +74,121 @@
+											for="turnTimeOpt6" class="draggable-label">&#8734;</label>
 									</div>
 								</div>
 								<div class="gameOption">
 									<h2>Time Pools [ minutes ]</h2>
 									<div id="timePool" class="typeRadio">
 										<!-- <input type="radio" id="timePoolOpt0" name="timePool" value="1" ><label for="timePoolOpt0">1 min</label> -->
-										<input type="radio" id="timePoolOpt1" name="timePool" value="15"><label for="timePoolOpt1">15</label>
-										<input type="radio" id="timePoolOpt2" name="timePool" value="20"><label for="timePoolOpt2">20</label>
-										<input type="radio" id="timePoolOpt3" name="timePool" value="25"><label for="timePoolOpt3">25</label>
-										<input type="radio" id="timePoolOpt4" name="timePool" value="30"><label for="timePoolOpt4">30</label>
+										<input type="radio" id="timePoolOpt1" name="timePool" value="15"><label for="timePoolOpt1" class="draggable-label">15</label>
+										<input type="radio" id="timePoolOpt2" name="timePool" value="20"><label for="timePoolOpt2" class="draggable-label">20</label>
+										<input type="radio" id="timePoolOpt3" name="timePool" value="25"><label for="timePoolOpt3" class="draggable-label">25</label>
+										<input type="radio" id="timePoolOpt4" name="timePool" value="30"><label for="timePoolOpt4" class="draggable-label">30</label>
 										<input type="radio" id="timePoolOpt5" name="timePool" value="-1" checked><label
-											for="timePoolOpt5">&#8734;</label>
+											for="timePoolOpt5" class="draggable-label">&#8734;</label>
 									</div>
 								</div>
 								<div class="gameOption">
 									<h2>Combat Location [ no influence to the actual gameplay ]</h2>
 									<div id="combatLocation" class="typeRadio">
-										<input type="radio" id="bgOpt1" name="combatLocation" value="Dark Forest"><label for="bgOpt1">Dark Forest</label>
-										<input type="radio" id="bgOpt2" name="combatLocation" value="Frozen Wall"><label for="bgOpt2">Frozen Wall</label>
-										<input type="radio" id="bgOpt3" name="combatLocation" value="Shadow Cave"><label for="bgOpt3">Shadow Cave</label>
-										<input type="radio" id="bgOpt4" name="combatLocation" value="Dragon Bones"><label for="bgOpt4">Dragon Bones</label>
+										<input type="radio" id="bgOpt1" name="combatLocation" value="Dark Forest"><label for="bgOpt1" class="draggable-label">Dark Forest</label>
+										<input type="radio" id="bgOpt2" name="combatLocation" value="Frozen Wall"><label for="bgOpt2" class="draggable-label">Frozen Wall</label>
+										<input type="radio" id="bgOpt3" name="combatLocation" value="Shadow Cave"><label for="bgOpt3" class="draggable-label">Shadow Cave</label>
+										<input type="radio" id="bgOpt4" name="combatLocation" value="Dragon Bones"><label for="bgOpt4" class="draggable-label">Dragon Bones</label>
 									</div>
 								</div>
+								<!--Script to add sliding to prematch option buttons-->
+								<script>
```

Comment:
Best to make index file smaller (as we even plan to split it into files). Would be best if script was separate file in `src/ui` and be imported using webpack or whatever.